ddtrace/_monkey.py

```diff
@@ -209,13 +261,13 @@ def on_import(hook):
                 versions = imported_module.get_versions()
                 for name, v in versions.items():
                     telemetry.telemetry_writer.add_integration(
-                        name, True, PATCH_MODULES.get(module) is True, "", version=v
+                        name, should_patch, PATCH_MODULES.get(module) is True, "", version=v
```

Comment:
it could be nice to get a message here on why we didn't patch.

Reply:
good idea, i'll add a message if we didn't patch